# Change the values according to your camera and setup

rectified: true

# number of frames in dataset only used for non-real-time
numOfFrames: 1101

# path to image folder ( if not real-time )
imagesPath: "/home/christos/catkin_ws/src/mini_project_kokas/src/vio_slam/images/kitti/06/"

# image extension
fileExtension: ".png"

# front camera params
Camera_l:
  fx: 707.0912
  fy: 707.0912
  cx: 601.8873
  cy: 183.1104
  k1: 0
  k2: 0
  p1: 0
  p2: 0
  k3: 0

Camera_r:
  fx: 707.0912
  fy: 707.0912
  cx: 601.8873
  cy: 183.1104
  k1: 0
  k2: 0
  p1: 0
  p2: 0
  k3: 0

Camera:
  width: 1241
  height: 376
  fps: 10.0
  bl: 0.53715

# Feature Extractor Values
FE:
  nFeatures: 2000
  nLevels: 8
  imScale: 1.2
  edgeThreshold: 19
  maxFastThreshold: 20
  minFastThreshold: 7
  patchSize: 31